### PR TITLE
[sysdig] update for tech preview release of Runtime Scanner

### PR DESCRIPTION
**Pull request by ProvoK**

### CODE REVIEW

File: charts/sysdig/values.yaml [hunk 1]
```diff
@@ -474,11 +476,11 @@ nodeAnalyzer:
     resources:
       requests:
         cpu: 150m
-        memory: 512Mi
+        memory: 1024Mi
```
Comment (blkt) [inline charts/sysdig/values.yaml]: Given that this is a DaemonSet, the option (1) of increasing requests could prevent scheduling of the pod if requests are unsatisfiable, while option (2) of having lower requests and high limits (i.e. `512Mi/2Gi`) guarantees the `helm install ...` command terminates and possibly only at a later time have problems finding enough memory to run.

Are we fine going with option (1)?

**Review by blkt — APPROVED** [review]
LGTM

File: charts/sysdig/values.yaml (hunk 1 above)
Comment (ProvoK) [reply charts/sysdig/values.yaml]: It's likely not gonna work - as of now - with 512Mi, improvements for memory usage are in the pipeline though.

But I see the point in preferring a single container OOMKilled than the entire Pod "Pending".
I'm rollbacking this change

Comment (ProvoK) [reply charts/sysdig/values.yaml]: (done)

**Review by blkt — APPROVED** [review]

**Pull request merged**